products/ome-files/downloads/index.html

```diff
@@ -100,8 +100,8 @@ <h3 class="text-center">Self-contained "super" builds</h3>
                     <div class="card-section">
                         <img alt="Windows logo" src="{{ site.baseurl }}/img/logos/3rd-party_windows.png">
                         <h5>VC14 32‑bit (VS2015)</h5>
```

Comment:
While you're editing this anyway, 'Windows' is missing from the beginning of this heading